cmd/registry/root.go

```diff
@@ -0,0 +1,261 @@
+package registry
+
+import (
+	"fmt"
+	"github.com/astronomer/astro-cli/pkg/input"
+	"github.com/logrusorgru/aurora"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+)
+
+var (
+	dagVersion      string
+	providerVersion string
+)
+
+func getProviderSearchRoute(providerName string) string {
+	getProviderUrl := url.URL{
+		Scheme:   "https",
+		Host:     registryHost,
+		Path:     fmt.Sprintf("%s/%s", registryApi, providerSearchRoute),
+		RawQuery: fmt.Sprintf(providerSearchQuery, url.QueryEscape(providerName)),
+	}
+	return getProviderUrl.String()
+}
+
+func getProviderRoute(providerId string, providerVersion string) string {
```

Comment:
```suggestion
func getProviderRoute(providerName string, providerVersion string) string {
```

Reply:
The airflow schema refers to _apache-airflow-provider-snowflake_ as a "provider package name" and "amazon" would be the "provider name". see here: https://github.com/apache/airflow/blob/main/airflow/provider_info.schema.json